### DISCUSSION TITLE
[help] compatibility with sweave/knitr ?

### DISCUSSION THREAD

**Discussion by jrdavalos** [ropensci/targets#1295] [Help] [answered]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

Hi, I'm using tar_render() at the end of my pipeline but I'm not satisfied with the rendering with Rmarkdown. I'm used to make LaTeX documents and I'd like to use Sweave/knitr at the end of the pipeline. Is this possible somehow?

**Comment by wlandau** [comment]
Please see `tar_knit()`: https://docs.ropensci.org/tarchetypes/reference/tar_knit.html

**Reply by jrdavalos** [reply]
Thanks, but there is something I don't understand. I have my .Rnw file and the pipeline worked (no errors), but there is no output at all. `tar_knit()` seems to work like `tar_render()` so I don't understand why it does that.

**Reply by wlandau** [reply, marked as answer]
Do you have a reprex?

On my end `tar_make()` produces a `file.tex` file with this pipeline:

```r
# _targets.R
library(targets)
library(tarchetypes)
list(
  targets::tar_target(data, data.frame(x = seq_len(26), y = letters)),
  tarchetypes::tar_knit(report, "file.Rnw")
)
```

file.Rnw:

```r
\documentclass{article}

\title{Minimal example}
\author{You}
\date{\today}

\begin{document}

\maketitle

Data:

<<minimal-example, echo=TRUE>>=
targets::tar_read(data)
@

\end{document}
```

**Reply by jrdavalos** [reply]
Well, after a closer look I found the .tex : it was in the main folder and not in the subfolder with the Rnw ! Thanks.
Last question : do you know if there is a way to execute the .tex inside the pipeline ?

**Reply by wlandau-lilly** [reply]
You could add a downstream target that depends on `report` and runs a LaTeX system call.
